dask/dataframe/tests/test_dataframe.py

```diff
@@ -3201,27 +3202,50 @@ def test_round():
     assert_eq(ddf.round(2), df.round(2))
 
 
-def test_cov():
-    # DataFrame
+@pytest.mark.parametrize(
+    "numeric_only",
+    [
+        None,
+        pytest.param(
+            True,
+            marks=pytest.mark.skipif(
+                not PANDAS_GT_150, reason="numeric_only not yet implemented"
+            ),
+        ),
+        pytest.param(
+            False,
+            marks=pytest.mark.skipif(
+                not PANDAS_GT_150, reason="numeric_only not yet implemented"
+            ),
+        ),
+    ],
+)
+def test_cov_dataframe(numeric_only):
     df = _compat.makeMissingDataframe()
     ddf = dd.from_pandas(df, npartitions=6)
 
-    res = ddf.cov()
-    res2 = ddf.cov(split_every=2)
-    res3 = ddf.cov(10)
-    res4 = ddf.cov(10, split_every=2)
-    sol = df.cov()
-    sol2 = df.cov(10)
+    numeric_only_kwarg = {}
+    if numeric_only is not None and PANDAS_GT_150:
```

Comment:
Wouldn't `pytest.mark.skipif` take care of it?

Reply:
You're correct, I've removed the unnecessary `and PANDAS_GT_150` part here (and in the other tests)